capstone/src/main/webapp/script.js

```diff
@@ -1,110 +1,21 @@
 function createMap() {
   const map = new google.maps.Map(
       document.getElementById('map-container'),
-      {center: {lat: 37.7749, lng: -122.4194}, zoom: 13});
-
+      {center: {lat: 37.7749, lng: -122.4194}, zoom: 12});
+  
 	    //**adding zipcode overlay */
         map.data.loadGeoJson('zipcode-data.json');
         map.data.setStyle({visible: false});
+      //**adding precinct overlay */
+        map.data.loadGeoJson('neighborhoods.json');
+        map.data.setStyle({visible: false});
+      {center: {lat: 37.7749, lng: -122.4194}, zoom: 13});
 
-    //**cordinates for shapes */
     var cityLimits = [
```

Comment:
Don't use var, switch to let or const instead

Reply:
var is how the Map API tutorial did it, so I think it should be okay

Reply:
I haven't seen the Map API tutorial but var is specifically against Google's style guides (https://google.github.io/styleguide/jsguide.html#features-local-variable-declarations)